viite-UI/manual/index.md

```diff
@@ -238,22 +239,22 @@ Pakollisia tietoja ovat nimi ja projektin muutosten voimaantulopäivämäärä,
 
 ![Uusi tieosoiteprojekti](k19.jpg)
 
-_kuva 6.2: Tieosoiteprojektin perustietojen täyttäminen._
+_Kuva 6.2: Tieosoiteprojektin perustietojen täyttäminen._
 
 Projektin tieosat lisätään täydentämällä niiden tiedot kenttiin TIE, AOSA sekä LOSA ja painamalla painiketta Varaa (6). __Kaikki kentät tulee täyttää, jos haluaa varata tieosan!__ 
 
 Varauksen yhteydessä järjestelmä tekee varattaville tieosille tarkastukset:
 
-- Onko varattava tieosa olemassa projektin voimaantulopäivänä
-- Onko varattava tieosa vapaana vai onko se jo varattu toiseen projektiin
+- Vrattavan tieosan täytyy olla olemassa projektin voimaantulopäivänä
```

Comment:
typo

Reply:
Fiksattu -> "Varattava" commitissa 32efe42 (Manual: Fixed PR comment, and spell check errors)